### PR TITLE
Feature/fix tests

### PR DESCRIPTION
**Pull request by JeremySkinner**

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -5,10 +5,6 @@ environment:
   CA_KEY:
     secure: JQtUaB+zduPID8bzI5nfHRc2V6xYJiJp4mTHkE4W6broH8rznT1ypvn5orQ6YcGn
 
-branches:
-  only:
-    - master
```
Comment (dahlbyk) [inline appveyor.yml]: We'll want to put this back after PR builds via webhook are working correctly.

File: test/Shared.ps1 [hunk 2]
```diff
@@ -29,7 +27,7 @@ function MakeNativePath([string]$Path) {
   $Path -replace '\\|/', [System.IO.Path]::DirectorySeparatorChar
 }
 
-[System.Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSUseDeclaredVarsMoreThanAssigments', '')]
-$module = Import-Module $PSScriptRoot\..\posh-sshell.psd1
+# [System.Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSUseDeclaredVarsMoreThanAssigments', '')]
+# $module = Import-Module $PSScriptRoot\..\posh-sshell.psd1
```
Comment (dahlbyk) [inline test/Shared.ps1]: Go ahead and delete this. Shouldn't be necessary since this module doesn't need posh-git's `prompt` stuff.

**Review by dahlbyk — APPROVED** [review]

File: appveyor.yml (hunk 1 above)
Comment (JeremySkinner) [reply appveyor.yml]: Thanks, I'll add that back in.

**Comment by dahlbyk** [comment]
> Add Linux build

I've never used Appveyor Linux support. How will this be reflected in status checks?

**Review by dahlbyk — APPROVED** [review]

**Comment by JeremySkinner** [comment]
It just appears as a second project in appveyor with a second webhook, so should just show as a second required status check on the PR page. You can then have a second badge that shows the result of the other build (Eg see https://github.com/jeremyskinner/fluentvalidation and https://ci.appveyor.com/project/JeremySkinner/fluentvalidation-tq12c)

If you don't think it's a good idea, we can leave it for now.

**Comment by dahlbyk** [comment]
Just added Linux webhook, too.

**Comment by JeremySkinner** [comment]
Thanks, the linux build is now passing. I had to make a couple of changes:

- Removed the install line `Install-PackageProvider -Name NuGet` as this fails and isn't necessary (I'm not sure if we need it on the windows version either?)
- Worked around coveralls install issue

Coveralls is registered on PS Gallery with the name "coveralls", but the module file is "Coveralls.psd1". The inconsistent casing means that `Import-Module` will fail as it can't find the module. I've hard-coded the path to the module for now, and I've opened an issue on the coveralls repo: https://github.com/JanDeDobbeleer/coveralls/issues/4

I haven't set up an account on coveralls.io yet for posh-sshell, I'll try and do that today.

**Pull request merged**